x-pack/plugins/security_solution/public/timelines/components/graph_overlay/translations.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const BACK_TO_EVENTS = i18n.translate(
+  'xpack.securitySolution.timeline.graphOverlay.backToEventsButton',
+  {
+    defaultMessage: '< Back to events',
```

Comment:
nit -> do we want to give the new name of the timeline here

Reply:
I don't think it makes sense to include the name of the timeline in the `< Back to events` button because the user is still "in" that timeline, and the Timeline name is always visible, per the screenshot below:

![case-integration](https://user-images.githubusercontent.com/4459398/85886773-e3587980-b7a3-11ea-87b6-b098ea14bc5f.png)